gaeguli/meson.build

```diff
@@ -45,6 +47,43 @@ libgaeguli = library(
   install: true
 )
 
+if meson.version().version_compare('>= 0.48.2')
+
+  pkg.generate(libgaeguli,
+    libraries : [ gst_dep, libsrt_dep ],
+    name : meson.project_name(),
+    description : 'A SRT Streaming library',
+    filebase : gaeguli_api_name,
+    subdirs: gaeguli_api_name,
+    variables: 'exec_prefix=${prefix}'
+  )
+
+else
+
+  pkg.generate(
+    libraries : libgaeguli,
+    version : meson.project_version(),
+    name : meson.project_name(),
+    description : 'A SRT Streaming library',
+    filebase : gaeguli_api_name,
+    requires : [
+      'gstreamer-1.0',
+      'gstreamer-base-1.0',
+      'srt'
```

Comment:
Does gaeguli only use gstreamer-1.0 and gstreamer-base-1.0?

Reply:
Currently, yes. but I'll update for consistency of build script.